sdk/resourcemanager/azure-resourcemanager-eventhubs/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.27.0-beta.1 (2023-04-23)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Updated core dependency from resources.
```

Comment:
Change log would be like "- Updated `api-version` to `2021-11-01`."

Reply:
Fixed in the new version

Reply:
Please, what api-version do you upgrade here?